proxy/backend/core/src/test/resources/expected/export-storage-nodes.json

```diff
@@ -1 +1 @@
-{"storage_nodes":[{"port":"0","password":"root","database":""}]}
\ No newline at end of file
+{"storage_nodes":{"normal_db":[{"port":"0","password":"root","database":""}]}}
```

Comment:
Please keep a blank line at the end of file.

Reply:
Thanks, I have fixed.